.flutter-plugins-dependencies

```diff
@@ -0,0 +1 @@
+{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"shared_preferences","path":"/Users/fks/.pub-cache/hosted/pub.dartlang.org/shared_preferences-2.0.8/","native_build":true,"dependencies":[]}],"android":[{"name":"shared_preferences","path":"/Users/fks/.pub-cache/hosted/pub.dartlang.org/shared_preferences-2.0.8/","native_build":true,"dependencies":[]}],"macos":[{"name":"shared_preferences_macos","path":"/Users/fks/.pub-cache/hosted/pub.dartlang.org/shared_preferences_macos-2.0.2/","native_build":true,"dependencies":[]}],"linux":[{"name":"path_provider_linux","path":"/Users/fks/.pub-cache/hosted/pub.dartlang.org/path_provider_linux-2.1.0/","native_build":false,"dependencies":[]},{"name":"shared_preferences_linux","path":"/Users/fks/.pub-cache/hosted/pub.dartlang.org/shared_preferences_linux-2.0.2/","native_build":false,"dependencies":["path_provider_linux"]}],"windows":[{"name":"path_provider_windows","path":"/Users/fks/.pub-cache/hosted/pub.dartlang.org/path_provider_windows-2.0.3/","native_build":false,"dependencies":[]},{"name":"shared_preferences_windows","path":"/Users/fks/.pub-cache/hosted/pub.dartlang.org/shared_preferences_windows-2.0.2/","native_build":false,"dependencies":["path_provider_windows"]}],"web":[{"name":"shared_preferences_web","path":"/Users/fks/.pub-cache/hosted/pub.dartlang.org/shared_preferences_web-2.0.2/","dependencies":[]}]},"dependencyGraph":[{"name":"path_provider_linux","dependencies":[]},{"name":"path_provider_windows","dependencies":[]},{"name":"shared_preferences","dependencies":["shared_preferences_linux","shared_preferences_macos","shared_preferences_web","shared_preferences_windows"]},{"name":"shared_preferences_linux","dependencies":["path_provider_linux"]},{"name":"shared_preferences_macos","dependencies":[]},{"name":"shared_preferences_web","dependencies":[]},{"name":"shared_preferences_windows","dependencies":["path_provider_windows"]}],"date_created":"2022-07-04 21:50:31.021760","version":"3.0.4"}
```

Comment:
files have been added. Are these all necessary files? If not, please remove them.